new/cookbook-abandonment.md

```diff
@@ -0,0 +1,50 @@
+---
+RFC: unassigned
+Author: Nell Shamrell-Harrington <nshamrell@chef.com>
+Status: Draft
+Type: Process
+---
+
+# Cookbook Abandonment On Supermarket
+
+The Chef community needs a process around handling when cookbooks on Supermarket are abandoned.  This occurs when a cookbook owner does not respond to requests to adopt the cookbook, and the cookbook has not been updated in some time.
+
```

Comment:
That's a very good question.  My inclination is yes, the is also intended to cover the case where the cookbook owner has passed away, which we have seen happen in the rubygems and other communities.  Looks like RubyGems gives access to someone on the Github commit list: http://help.rubygems.org/discussions/problems/17925-abandoned-gem-nominee-for-new-maintainer